src/powershell/install.sh

```diff
@@ -221,25 +221,31 @@ install_using_github() {
         echo "${powershell_archive_sha256} *${powershell_filename}" | sha256sum -c -
     fi
     tar xf "${powershell_filename}" -C "${powershell_target_path}"
-    ln -s "${powershell_target_path}/pwsh" /usr/local/bin/pwsh
```

Comment:
Should we keep this symbolic linking intact? I am afraid if folks might be relying on this.

Reply:
it is down in line 225, with added "force" to overwrite any potentially existing symlink. The reason for that is that otherwise, when other features use `dependsOn` to refer to the PowerShell feature, and they also explicitly define the PowerShell feature to install modules, it does not play nice with existing symlinks, leading to a build failure.